nixos/modules/services/web-apps/zabbix.nix

```
@@ -280,6 +280,8 @@ in
     services.phpfpm.pools.zabbix = {
       inherit user;
       group = config.services.${cfg.frontend}.group;
+      phpPackage =
+        if cfg.package == pkgs.zabbix.web then pkgs.php83 else config.services.phpfpm.phpPackage;
```

```suggestion
      phpPackage =
        if cfg.package == pkgs.zabbix.web then pkgs.php83 else pkgs.php84;
```
to not ugprade to 8.5 on accident